apps/web/src/pages/templates/editor/LocaleSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import { SelectItemProps, Group } from '@mantine/core';
+import { Select, Text } from '@novu/design-system';
+import { forwardRef } from 'react';
+import { IS_DOCKER_HOSTED } from '../../../config';
+
+export function LocaleSelect({ locales, value, isLoading, readonly, setSelectedLocale }) {
```

Comment:
duplicated component? `apps/web/src/components/workflow/Preview/common/LocaleSelect.tsx`

Reply:
I think these were left behind after the rebase, coz i moved these components to the newly created common folder.
Will delete these